.github/workflows/pre-commit-lint-check.yaml

```diff
@@ -10,4 +10,3 @@ jobs:
       - run: npm install
       - run: npm run lint-check
       - run: npm run format-check
-      - run: npm test
```

Comment:
Do we not need tests in workflow

Reply:
yes, if i don't run test in local and create a pr so it works as a fail-safe the tests will run in PRs

Reply:
But in this change you're removing the step that runs the test in pr. Pls correct me if I am wrong